### PR TITLE
Bug Fix Running CI Integration Test

### PR DESCRIPTION
**Pull request by MobiTikula**
Fix the bug, so it is possible to run Liv Doc on a CI.

#22

### CODE REVIEW

**Comment by MobiTikula** [comment]
There was a problem that CI run did not fetch extra project data.

The bug was caused by limited scope of the GitHub token. There is a need to add a LIV_DOC_ACCESS_TOKEN into the GH secrets, that are stored in the repository (the repo, from which we run the LIV DOC Action).
Currently workflow solution can be seen here: https://github.com/absa-group/living-doc-example-project/actions/runs/10699120043/workflow.

**Review by miroslavpojer — APPROVED** [review]

**Comment by miroslavpojer** [comment]
I approved the latest changes. The bug solution still needs to be talked about.

File: README.md [hunk 1]
```diff
@@ -113,15 +113,22 @@ See the full example of action step definition (in example are used non-default
...
+      GITHUB_TOKEN: ${{ secrets.LIV_DOC_ACCESS_TOKEN }}
     ```
     
+#### How to Store Token as a Secret
+1. Go to the GitHub repository, from which you want to run the GitHub Action.
+2. Click on the `Settings` tab in the top bar.
+3. In the left sidebar, click on `Secrets and variables` > `Actions`.
+4. Click on the `New repository secret` button.
+5. Name the token `LIV_DOC_ACCESS_TOKEN` and paste the token value.
+
```
Comment (miroslavpojer) [inline README.md:131]: I am missing hint how to create and setup the token.

Comment (MobiTikula) [reply README.md:131]: Thank you for your research! I will add the directions into the README file.

**Comment by miroslavpojer** [comment]
> Release Notes Bug fix for correct CI run. Integration CI test successful.

use bullet points for release notes lines, please

**Review by miroslavpojer — APPROVED** [review]

**Pull request merged**